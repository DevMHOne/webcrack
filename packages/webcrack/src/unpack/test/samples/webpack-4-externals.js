(window['webpackJsonp'] = window['webpackJsonp'] || []).push([
  [],
  [
    function (module, exports, __webpack_require__) {
      var path = __webpack_require__(1);
      console.log(path);
    },
    function (module, exports) {
      module.exports = require('path');
    },
  ],
]);
