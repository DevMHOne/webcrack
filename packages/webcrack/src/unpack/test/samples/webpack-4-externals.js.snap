WebpackBundle {
  "chunks": [
    WebpackChunk {
      "chunkIds": [],
      "entryIds": [],
      "modules": Map {
        "0" => WebpackModule {
          "ast": import * as path from "path";
console.log(path);,
          "dependencies": Set {
            "1",
          },
          "exports": Set {},
          "externalModule": undefined,
          "id": "0",
          "isEntry": false,
          "path": "0.js",
        },
        "1" => WebpackModule {
          "ast": module.exports = require("path");,
          "dependencies": Set {},
          "exports": Set {},
          "externalModule": "path",
          "id": "1",
          "isEntry": false,
          "path": "1.js",
        },
      },
    },
  ],
  "entryId": "",
  "modules": Map {
    "0" => WebpackModule {
      "ast": import * as path from "path";
console.log(path);,
      "dependencies": Set {
        "1",
      },
      "exports": Set {},
      "externalModule": undefined,
      "id": "0",
      "isEntry": false,
      "path": "0.js",
    },
    "1" => WebpackModule {
      "ast": module.exports = require("path");,
      "dependencies": Set {},
      "exports": Set {},
      "externalModule": "path",
      "id": "1",
      "isEntry": false,
      "path": "1.js",
    },
  },
  "type": "webpack",
}